.rspec

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
```

Comment:
when you `--require rails_helper` here, you can omit it in every test file

Reply:
My typical `.rspec` file is 
```
--require rails_helper
--format documentation
--color
--order defined
```

Reply:
> My typical `.rspec` file is
> 
> ```
> --require rails_helper
> --format documentation
> --color
> --order defined
> ```

I knew I was forgetting something

Reply:
> I knew I was forgetting something

You weren't forgetting something. It's a matter of what you want or don't want. Especially `--order defined` is purely taste. I like to know when which test is executed, for others it is more important that the order of the tests is randomized, so they can be sure, that the green test does not rely on previously run tests.

Reply:
I like it to, makes the output easy to read.